src/docker-in-docker/install.sh

```diff
@@ -10,7 +10,7 @@
 
 DOCKER_VERSION="${VERSION:-"latest"}" # The Docker/Moby Engine + CLI should match in version
 USE_MOBY="${MOBY:-"true"}"
```

Comment:
Can you bump patch versions for both of the updated Features?

Reply:
Done! https://github.com/devcontainers/features/pull/867/commits/55f07186eb973aaafdd13d13dbb59fb4ba990dd5